### PR TITLE
Changed code references to reflect code changes

### PR DESCRIPTION
**Pull request by rpetrusha**
## Changed code references to reflect code changes

Related to #590
Contributes to dotnet/docs#2994

### CODE REVIEW

**Comment by rpetrusha** [comment]
Closing and reopening to begin new build after dotnet/samples#590 is merged.

**Comment by rpetrusha** [comment]
Closing and reopening to begin new build.

File: docs/framework/wpf/app-development/application-management-overview.md [hunk 1]
```diff
@@ -50,10 +50,10 @@ All applications tend to share a common set of functionality that applies to app
   
 <a name="The_Application_Definition"></a>   
 ## The Application Definition  
- To utilize the functionality of the <xref:System.Windows.Application> class, you must implement an application definition. A [!INCLUDE[TLA2#tla_wpf](../../../../includes/tla2sharptla-wpf-md.md)] application definition is a class that derives from <xref:System.Windows.Application> and is configured with a special [!INCLUDE[TLA#tla_msbuild](../../../../includes/tlasharptla-msbuild-md.md)] setting.  
-  
+ To utilize the functionality of the <xref:System.Windows.Application> class, you must implement an application definition. A WPF application definition is a class that derives from <xref:System.Windows.Application> and is configured with a special Microsoft build engine (MSBuild) setting.
```
Comment (mairaw) [inline docs/framework/wpf/app-development/application-management-overview.md]: Do we really need to spell out MSBuild?

File: docs/framework/wpf/app-development/application-management-overview.md [hunk 2]
```diff
@@ -64,19 +64,19 @@ All applications tend to share a common set of functionality that applies to app
...
   
--   In markup, the `Application` element must include the `x:Class` attribute. When the application is built, the existence of `x:Class` in the markup file causes [!INCLUDE[TLA2#tla_msbuild](../../../../includes/tla2sharptla-msbuild-md.md)] to create a `partial` class that derives from <xref:System.Windows.Application> and has the name that is specified by the `x:Class` attribute. This requires the addition of an [!INCLUDE[TLA2#tla_xml](../../../../includes/tla2sharptla-xml-md.md)] namespace declaration for the [!INCLUDE[TLA2#tla_xaml](../../../../includes/tla2sharptla-xaml-md.md)] schema ( `xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"` ).  
+-   In markup, the `Application` element must include the `x:Class` attribute. When the application is built, the existence of `x:Class` in the markup file causes MSBuild to create a `partial` class that derives from <xref:System.Windows.Application> and has the name that is specified by the `x:Class` attribute. This requires the addition of an XML namespace declaration for the XAML schema ( `xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"` ).
```
Comment (mairaw) [inline docs/framework/wpf/app-development/application-management-overview.md]:
```suggestion
- In markup, the `Application` element must include the `x:Class` attribute. When the application is built, the existence of `x:Class` in the markup file causes MSBuild to create a `partial` class that derives from <xref:System.Windows.Application> and has the name that is specified by the `x:Class` attribute. This requires the addition of an XML namespace declaration for the XAML schema (`xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"`).
```

File: docs/framework/wpf/app-development/application-management-overview.md [hunk 3]
```diff
@@ -64,19 +64,19 @@ All applications tend to share a common set of functionality that applies to app
...
+ This code is the minimum that is required to implement an application definition. However, an additional MSBuild configuration needs to be made to the application definition before building and running the application.  
   
 ### Configuring the Application Definition for MSBuild  
- Standalone applications and [!INCLUDE[TLA#tla_xbap#plural](../../../../includes/tlasharptla-xbapsharpplural-md.md)] require the implementation of a certain level of infrastructure before they can run. The most important part of this infrastructure is the entry point. When an application is launched by a user, the operating system calls the entry point, which is a well-known function for starting applications.  
+ Standalone applications and XBAPs require the implementation of a certain level of infrastructure before they can run. The most important part of this infrastructure is the entry point. When an application is launched by a user, the operating system calls the entry point, which is a well-known function for starting applications.
```
Comment (mairaw) [inline docs/framework/wpf/app-development/application-management-overview.md]: would it be good to continue to spell out XBAP here?

**Review by mairaw — APPROVED** [review]
Looks great @rpetrusha! Left some comments for you to consider.

**Pull request merged**